jobs/mesoscale/plots/JEVS_MESOSCALE_PLOTS

```diff
@@ -5,41 +5,42 @@
 ####################################
 set -xa
 
+export SENDCOM=${SENDCOM:-YES}
+export SENDDBN=${SENDDBN:-YES}
+export SENDECF=${SENDECF:-YES}
+export SENDDBN_NTC=${SENDDBN_NTC:-NO}
 
 ####################################
 # SET SHELL PROCESSING VARIABLES
 ####################################
 export PS4='$SECONDS + '
 date
 
-
 ####################################
 # obtain unique LSF id (jobid) and make temp directories
 ####################################
 export pid=$$
 export DATA=${DATA:-${DATAROOT:?}/${jobid:?}}
-rm -rf $DATA
 mkdir -p $DATA
 cd $DATA
 
 
-####################################
-# Determine Job Output Name on System
-####################################
-export pgmout="OUTPUT.${pid}"
-
-
 ####################################
 # Set EVS directories
 ####################################
+export machine=${machine:-WCOSS2}
+export MET_bin_exec=${MET_bin_exec:-bin}
```

Comment:
This line (32) is not needed as it is defined in `parm/metplus_config/machine.conf`